### ISSUE TITLE
Failing links from website to most FIBO Q1 2020 Production Release files 

### ISSUE THREAD

**Issue by rivettp** [edmcouncil/fibo#967]
I did not try all links but all the PROD files for 2020Q1, which I tried, which are linked to from the live website, failed to produce a download but redirected to a more general page, in some cases with an error message at the top of the page "Error! Cannot fetch data, please try later."

OWL: https://spec.edmcouncil.org/fibo/ontology/master/2020Q1/prod.rdf.zip 
SKOS from: https://spec.edmcouncil.org/fibo/vocabulary
Glossary CSV and Excel from here: https://spec.edmcouncil.org/fibo/data-dictionary 
Quickstart: https://spec.edmcouncil.org/fibo/ontology/master/2020Q1/prod.fibo-quickstart.ttl

This was the case last night when I noticed a link didn't work, and assumed it was a temporary deployment glitch - but it remains the case now and across all links (I tried a different browser too). This is especially embarrassing due to the launch of the site today!
We should ensure a coordinated release rather than the links on the website being updated before the files are actually available. And the site should be reviewed and validated!

**Comment by jziemer2** [comment]
Thanks, Pete, for opening the issue.
I tried all FIBO OWL Production links, prod.rdf.zip, prod.ttl.zip, prod.jsonld.zip, prod.fibo.nqprod.fibo.nq.zip with the same error.

**Comment by trypuz** [comment]
Fixed.

**Comment by rivettp** [comment]
Regardless of the root cause in this case, I'd also like to see a fix to the **process** (ideally automated but in the short term manual is OK) to ensure we apply reasonable and organized testing that our primary products are properly accessible via the site; and it's not just left to chance (I just happened to be looking at something else) that such major problems are discovered.

This is not the first time we've had our major products inaccessible (see INFRA-477 back in December) and I raised then the need for testing releases - there is a systemic issue here (and technical debt) we should not keep sweeping under the rug having fixed the immediate problem. It undermines our claims of having a robust and mature publication process.
